src/content/docs/full-stack-observability/observe-everything/get-started/new-relic-guided-install-overview.mdx

```diff
@@ -173,4 +173,53 @@ Our open source integrations send performance metrics and inventory data from yo
 			<td>newrelic install -n varnish-cache-open-source-integration</td>
 		</tr>
 	</tbody>
-</table>
\ No newline at end of file
+</table>
+
+## Troubleshoot common problems
+
+Guided install is fairly technically complex. As we identify areas where the guided install fails, we'll document them here and provide some guidance.
+
+### MySQL: Incorrect user permissions
```

Comment:
Maybe we could use a collapser here for each section (MySQL and NGINX).

Reply:
Maybe!